### PR TITLE
Improve tests codebase

### PR DESCRIPTION
**Pull request by Jeckerson**

### CODE REVIEW

File: tests/integration/Migration/Action/GenerateCest.php [hunk 1]
```diff
@@ -216,4 +216,68 @@ public function getQuoteWrappedColumns(IntegrationTester $I): void
...
+            $columns = [
+                new Column('unknown', [
+                    'type' => 9000,
+                    'size' => 10,
+                    'notNull' => true,
+                ]),
+            ];
+
+            $data = [];
+            $class = new Generate('mysql', $columns);
```
Comment (jenovateurs) [inline tests/integration/Migration/Action/GenerateCest.php]: Can you add PostgreSQL tests and maybe SQLite ?

File: tests/integration/Migration/Action/GenerateCest.php [hunk 2]
```diff
@@ -216,4 +216,68 @@ public function getQuoteWrappedColumns(IntegrationTester $I): void
...
+                'size' => 10,
+                'notNull' => true,
+                'autoIncrement' => true,
+                'first' => true,
+                'primary' => true,
+                'default' => 0,
+            ]),
+        ];
+
+        $class1 = new Generate('mysql', $columnsWithDefault);
```
Comment (jenovateurs) [inline tests/integration/Migration/Action/GenerateCest.php]: Same things

File: tests/integration/Migration/Action/GenerateCest.php (hunk 1 above)
Comment (Jeckerson) [reply tests/integration/Migration/Action/GenerateCest.php]: Here it is not DB test, but class(es) logic.

File: tests/integration/Migration/Action/GenerateCest.php (hunk 2 above)
Comment (Jeckerson) [reply tests/integration/Migration/Action/GenerateCest.php]: Here it is not DB test, but class(es) logic.

File: tests/integration/Migration/Action/GenerateCest.php (hunk 1 above)
Comment (jenovateurs) [reply tests/integration/Migration/Action/GenerateCest.php]: But after, create a new Generate instance, you use getColumns(). 
Then getColumns() use getColumnSize() and in this particular function there is a difference between Postgres and Mysql.
`if ($this->adapter === Migration::DB_ADAPTER_POSTGRESQL && in_array($columnType, $noSizePostgres))`
I think it's useful to test it to check if there is no error produce.

Comment (Jeckerson) [reply tests/integration/Migration/Action/GenerateCest.php]: Added for all 3 DB types.

**Pull request merged**